src/pkg/cli/compose/validation.go

```diff
@@ -28,14 +27,22 @@ func (e ErrMissingConfig) Error() string {
 	return fmt.Sprintf("missing configs %q (https://docs.defang.io/docs/concepts/configuration)", ([]string)(e))
 }
 
+type ErrManagedStoreParam string
+
+func (e ErrManagedStoreParam) Error() string {
+	return string(e)
+}
+
+type PostgresProps map[string]any
```

Comment:
You're not using this? Or did you intend it as `type PostgresProps map[string]any`